sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -2674,11 +2679,24 @@ object Sequence {
            |${genSequenceLengthCode(ctx, startMicros, stopMicros, intervalInMicros, arrLength)}
           """.stripMargin
 
+      val check = if (scale == MICROS_PER_DAY) {
+        s"""
+           |if ($stepMonths == 0 && $stepDays == 0) {
+           |   throw new IllegalArgumentException(
+           |     "sequence step must be a day interval if start and end values are dates");
```

Comment:
Sorry for nit-picking again. But line 2685 has 3 spaces instead of 2 spaces and line 2686 has 5 spaces instead of 4 spaces.